packages/flame/lib/src/components/position_component.dart

```diff
@@ -98,6 +102,12 @@ class PositionComponent extends Component
   final NotifyingVector2 _size;
   Anchor _anchor;
 
+  /// Should be used to account for any angular offset needed to
+  /// orient the component in desired direction. It can be thought
+  /// of as rotating the local co-ordinate system of the component
+  /// by [angularOffset].
+  double angularOffset;
```

Comment:
Not sure I like this name either, naming is hard! 😅
This should state which direction the component is looking at from the beginning, so the dartdoc is a bit wrong, it shouldn't do any modifications to the local co-ordinate system.

Reply:
Changed it to `nativeAngle` as suggested in #1107